READMEV2.md

```diff
@@ -0,0 +1,76 @@
+# VEDA Features API
+
+Hosting and serving collections of vector data features for VEDA
+
+---
+
+## Implementation
+
+* Storage: PostGIS
+* WFS3 API with query support: [OGC API Features](https://ogcapi.ogc.org/features/) provided by [TiPG](https://github.com/developmentseed/tipg)
+* Vector tiles API provided by [TiPG](https://github.com/developmentseed/tipg)
+
+---
+
+### Local Development in Docker
+
+To locally run the site:
+
+`docker-compose up`
```

Comment:
This will use the docker compose that references the old CDK stack. Will need to update to use `veda-wfs3-app`

Reply:
oh, interesting, I didn't know think that it referenced CDK at all, let me have a look and change things around. Good catch